[00:29:21,972 | DEBUG |     main |    57]: <module>() : -----------------Starting Log Session-----------------
[00:29:21,972 | DEBUG |     main |    58]: <module>() :                  16/10/2024, 00:29:21                 
[00:29:22,754 |  INFO |     main |    85]: checkScriptJobs() : scriptJobs created for "main" UI!
[00:29:24,101 | ERROR |    utils |   110]: warningInView() : Select one curve and one geometry or Select one target curve and any number of curveCards/Tubes
[00:29:24,105 | DEBUG |    utils |   112]: warningInView() : File "D:\P4/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\core.py", line 1359, in bind
[00:39:44,997 | DEBUG |     main |    57]: <module>() : -----------------Starting Log Session-----------------
[00:39:44,997 | DEBUG |     main |    58]: <module>() :                  16/10/2024, 00:39:44                 
[00:39:45,779 |  INFO |     main |    85]: checkScriptJobs() : scriptJobs created for "main" UI!
[01:21:13,543 | DEBUG |     main |    57]: <module>() : -----------------Starting Log Session-----------------
[01:21:13,543 | DEBUG |     main |    58]: <module>() :                  16/10/2024, 01:21:13                 
[01:21:14,324 |  INFO |     main |    85]: checkScriptJobs() : scriptJobs created for "main" UI!
[17:46:40,989 | DEBUG |     main |    57]: <module>() : -----------------Starting Log Session-----------------
[17:46:40,990 | DEBUG |     main |    58]: <module>() :                  16/10/2024, 17:46:40                 
[17:46:41,768 |  INFO |     main |    85]: checkScriptJobs() : scriptJobs created for "main" UI!
[18:04:19,264 | DEBUG |     main |    57]: <module>() : -----------------Starting Log Session-----------------
[18:04:19,265 | DEBUG |     main |    58]: <module>() :                  16/10/2024, 18:04:19                 
[18:04:20,032 |  INFO |     main |    85]: checkScriptJobs() : scriptJobs created for "main" UI!
[18:11:08,557 | DEBUG |     main |    57]: <module>() : -----------------Starting Log Session-----------------
[18:11:08,558 | DEBUG |     main |    58]: <module>() :                  16/10/2024, 18:11:08                 
[18:11:09,340 |  INFO |     main |    85]: checkScriptJobs() : scriptJobs created for "main" UI!
[20:45:08,653 | DEBUG |     main |    57]: <module>() : -----------------Starting Log Session-----------------
[20:45:08,653 | DEBUG |     main |    58]: <module>() :                  16/10/2024, 20:45:08                 
[20:45:09,434 |  INFO |     main |    85]: checkScriptJobs() : scriptJobs created for "main" UI!
[01:15:01,809 | DEBUG |     main |    57]: <module>() : -----------------Starting Log Session-----------------
[01:15:01,809 | DEBUG |     main |    58]: <module>() :                  17/10/2024, 01:15:01                 
[01:15:02,494 |  INFO |     main |    85]: checkScriptJobs() : scriptJobs created for "main" UI!
[01:22:19,132 | DEBUG |     main |    57]: <module>() : -----------------Starting Log Session-----------------
[01:22:19,132 | DEBUG |     main |    58]: <module>() :                  17/10/2024, 01:22:19                 
[01:22:19,848 |  INFO |     main |    85]: checkScriptJobs() : scriptJobs created for "main" UI!
[01:23:59,953 | ERROR |    utils |   110]: warningInView() : Select at least one compatible card or tube curve.
[01:23:59,957 | DEBUG |    utils |   112]: warningInView() : File "H:\Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\core.py", line 4447, in subdivideCurve
[02:34:49,947 | DEBUG |     main |    57]: <module>() : -----------------Starting Log Session-----------------
[02:34:49,947 | DEBUG |     main |    58]: <module>() :                  17/10/2024, 02:34:49                 
[02:34:50,652 |  INFO |     main |    85]: checkScriptJobs() : scriptJobs created for "main" UI!
[02:34:53,204 | ERROR |    utils |   218]: wrapper() : 'NoneType' object is not iterable
Traceback (most recent call last):
  File "H:\Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\utils\utils.py", line 209, in wrapper
    rv = func(*args, **kwargs)
  File "H:\Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\core.py", line 5464, in extractAllCurves
    parentGrps = set(mc.listRelatives(allGeo, p=1))
TypeError: 'NoneType' object is not iterable
[03:23:13,056 | DEBUG |     main |    57]: <module>() : -----------------Starting Log Session-----------------
[03:23:13,056 | DEBUG |     main |    58]: <module>() :                  17/10/2024, 03:23:13                 
[03:23:13,807 |  INFO |     main |    85]: checkScriptJobs() : scriptJobs created for "main" UI!
[03:50:36,765 | DEBUG |     main |    57]: <module>() : -----------------Starting Log Session-----------------
[03:50:36,766 | DEBUG |     main |    58]: <module>() :                  17/10/2024, 03:50:36                 
[03:50:37,479 |  INFO |     main |    85]: checkScriptJobs() : scriptJobs created for "main" UI!
[04:12:37,267 | DEBUG |     main |    57]: <module>() : -----------------Starting Log Session-----------------
[04:12:37,267 | DEBUG |     main |    58]: <module>() :                  17/10/2024, 04:12:37                 
[04:12:37,980 |  INFO |     main |    85]: checkScriptJobs() : scriptJobs created for "main" UI!
[04:12:40,778 |  INFO |     core |  4046]: saveOptions() : Saving options
[04:12:53,200 |  INFO |     core |  4046]: saveOptions() : Saving options
[04:12:58,915 | ERROR |    utils |   103]: warning() : Invalid scale factor. Reverting to default (1.0).
[04:12:58,917 | DEBUG |    utils |   105]: warning() : File "H:\Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\core.py", line 4122, in getScaleFactor
[04:12:58,917 | ERROR |    utils |   110]: warningInView() : Select at least two curves
[04:12:58,918 | DEBUG |    utils |   112]: warningInView() : File "H:\Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\core.py", line 629, in populate
[04:13:01,958 | ERROR |    utils |   103]: warning() : Invalid scale factor. Reverting to default (1.0).
[04:13:01,958 | DEBUG |    utils |   105]: warning() : File "H:\Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\core.py", line 4122, in getScaleFactor
[04:13:01,959 | ERROR |    utils |   110]: warningInView() : Select at least two curves
[04:13:01,959 | DEBUG |    utils |   112]: warningInView() : File "H:\Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\core.py", line 629, in populate
[04:13:24,033 | ERROR |    utils |   103]: warning() : Invalid scale factor. Reverting to default (1.0).
[04:13:24,034 | DEBUG |    utils |   105]: warning() : File "H:\Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\core.py", line 4122, in getScaleFactor
[04:13:24,034 | ERROR |    utils |   110]: warningInView() : Select at least two curves
[04:13:24,035 | DEBUG |    utils |   112]: warningInView() : File "H:\Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\core.py", line 629, in populate
[04:13:25,792 |  INFO |     core |  4046]: saveOptions() : Saving options
[04:13:27,234 |  INFO |     core |  4046]: saveOptions() : Saving options
[04:13:28,198 | ERROR |    utils |   103]: warning() : Invalid scale factor. Reverting to default (1.0).
[04:13:28,199 | DEBUG |    utils |   105]: warning() : File "H:\Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\core.py", line 4122, in getScaleFactor
[04:13:28,199 | ERROR |    utils |   110]: warningInView() : Select at least two curves
[04:13:28,200 | DEBUG |    utils |   112]: warningInView() : File "H:\Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\core.py", line 629, in populate
[04:13:32,534 | ERROR |    utils |   103]: warning() : Invalid scale factor. Reverting to default (1.0).
[04:13:32,535 | DEBUG |    utils |   105]: warning() : File "H:\Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\core.py", line 4122, in getScaleFactor
[04:13:32,535 | ERROR |    utils |   110]: warningInView() : Select at least two curves
[04:13:32,536 | DEBUG |    utils |   112]: warningInView() : File "H:\Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\core.py", line 629, in populate
[04:13:34,020 | ERROR |    utils |   103]: warning() : Invalid scale factor. Reverting to default (1.0).
[04:13:34,021 | DEBUG |    utils |   105]: warning() : File "H:\Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\core.py", line 4122, in getScaleFactor
[04:13:47,269 |  INFO |     core |  4046]: saveOptions() : Saving options
[04:13:48,518 | ERROR |    utils |   103]: warning() : Invalid scale factor. Reverting to default (1.0).
[04:13:48,518 | DEBUG |    utils |   105]: warning() : File "H:\Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\core.py", line 4122, in getScaleFactor
[04:14:13,015 |  INFO |     core |  4046]: saveOptions() : Saving options
[04:14:13,016 | ERROR |       ui |  1465]: updateColors() : eval() arg 1 must be a string, bytes or code object
Traceback (most recent call last):
  File "H:\Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\ui.py", line 1456, in updateColors
    eval(mc.optionVar(q="GSCT_UVEditorBGColor")),
TypeError: eval() arg 1 must be a string, bytes or code object
[11:31:58,097 | DEBUG |     main |    57]: <module>() : -----------------Starting Log Session-----------------
[11:31:58,097 | DEBUG |     main |    58]: <module>() :                  17/10/2024, 11:31:58                 
[11:31:58,873 |  INFO |     main |    85]: checkScriptJobs() : scriptJobs created for "main" UI!
[15:56:12,030 | DEBUG |     main |    57]: <module>() : -----------------Starting Log Session-----------------
[15:56:12,030 | DEBUG |     main |    58]: <module>() :                  17/10/2024, 15:56:12                 
[15:56:12,833 |  INFO |     main |    85]: checkScriptJobs() : scriptJobs created for "main" UI!
<<<<<<< HEAD
[19:22:20,312 | DEBUG |     main |    57]: <module>() : -----------------Starting Log Session-----------------
[19:22:20,312 | DEBUG |     main |    58]: <module>() :                  17/10/2024, 19:22:20                 
[19:22:21,115 |  INFO |     main |    85]: checkScriptJobs() : scriptJobs created for "main" UI!
[01:36:19,841 | DEBUG |     main |    57]: <module>() : -----------------Starting Log Session-----------------
[01:36:19,841 | DEBUG |     main |    58]: <module>() :                  18/10/2024, 01:36:19                 
[02:06:24,324 | DEBUG |     main |    57]: <module>() : -----------------Starting Log Session-----------------
[02:06:24,324 | DEBUG |     main |    58]: <module>() :                  18/10/2024, 02:06:24                 
[02:06:25,046 |  INFO |     main |    85]: checkScriptJobs() : scriptJobs created for "main" UI!
[02:06:31,246 |  INFO |     core |  4059]: saveOptions() : Saving options
[02:06:31,797 | ERROR |    utils |   103]: warning() : Invalid scale factor. Reverting to default (1.0).
[02:06:31,804 | DEBUG |    utils |   105]: warning() : File "H:\Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\core.py", line 4135, in getScaleFactor
[02:06:31,807 | ERROR |    utils |   110]: warningInView() : Select at least one curve
[02:06:31,807 | DEBUG |    utils |   112]: warningInView() : File "H:\Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\core.py", line 563, in multiple
[02:06:32,506 | ERROR |    utils |   103]: warning() : Invalid scale factor. Reverting to default (1.0).
[02:06:32,508 | DEBUG |    utils |   105]: warning() : File "H:\Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\core.py", line 4135, in getScaleFactor
[02:06:34,114 | ERROR |    utils |   103]: warning() : Invalid scale factor. Reverting to default (1.0).
[02:06:34,116 | DEBUG |    utils |   105]: warning() : File "H:\Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\core.py", line 4135, in getScaleFactor
[02:06:34,135 |  INFO |     core |   579]: multiple() : pathCurve1 is not a compatible curve. Skipped.
[02:06:41,502 |  INFO |    utils |   118]: printInView() : Regrouped 1 curve(s)
[02:06:42,929 | ERROR |    utils |   110]: warningInView() : Select at least one Curve
[02:06:42,931 | DEBUG |    utils |   112]: warningInView() : File "H:\Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\core.py", line 6068, in groupCurves
[02:15:17,186 | ERROR |    utils |   110]: warningInView() : Select at least one Curve
[02:15:17,191 | DEBUG |    utils |   112]: warningInView() : File "H:\Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\core.py", line 5242, in resetCurvePivotPoint
[02:15:40,528 | ERROR |    utils |   110]: warningInView() : Select Edges to Convert
[02:15:40,530 | DEBUG |    utils |   112]: warningInView() : File "H:\Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\core.py", line 6175, in edgeToCurve
[02:15:48,006 | ERROR |    utils |   110]: warningInView() : Curve "polyToCurve1" failed to extract. Check the outliner for any loose curves.
[02:15:48,009 | DEBUG |    utils |   112]: warningInView() : File "H:\Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\core.py", line 5380, in extractSelectedCurves
[02:15:53,048 | ERROR |    utils |   103]: warning() : Invalid scale factor. Reverting to default (1.0).
[02:15:53,051 | DEBUG |    utils |   105]: warning() : File "H:\Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\core.py", line 4135, in getScaleFactor
[02:18:30,215 | DEBUG |     main |    57]: <module>() : -----------------Starting Log Session-----------------
[02:18:30,215 | DEBUG |     main |    58]: <module>() :                  18/10/2024, 02:18:30                 
[02:21:02,625 | DEBUG |     main |    57]: <module>() : -----------------Starting Log Session-----------------
[02:21:02,625 | DEBUG |     main |    58]: <module>() :                  18/10/2024, 02:21:02                 
[02:21:03,534 |  INFO |     main |    85]: checkScriptJobs() : scriptJobs created for "main" UI!
[02:26:30,792 | DEBUG |     main |    57]: <module>() : -----------------Starting Log Session-----------------
[02:26:30,792 | DEBUG |     main |    58]: <module>() :                  18/10/2024, 02:26:30                 
[02:31:45,489 | DEBUG |     main |    57]: <module>() : -----------------Starting Log Session-----------------
[02:31:45,489 | DEBUG |     main |    58]: <module>() :                  18/10/2024, 02:31:45                 
[02:31:46,329 |  INFO |     main |    85]: checkScriptJobs() : scriptJobs created for "main" UI!
[02:31:48,085 |  INFO |     core |  4048]: saveOptions() : Saving options
[02:31:51,269 | ERROR |    utils |   218]: wrapper() : 'NoneType' object is not iterable
Traceback (most recent call last):
  File "H:\Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\utils\utils.py", line 209, in wrapper
    rv = func(*args, **kwargs)
  File "H:\Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\core.py", line 5466, in extractAllCurves
    parentGrps = set(mc.listRelatives(allGeo, p=1))
TypeError: 'NoneType' object is not iterable
[02:32:34,507 | ERROR |    utils |   110]: warningInView() : Select at least one Curve
[02:32:34,508 | DEBUG |    utils |   112]: warningInView() : File "H:\Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\core.py", line 5353, in extractSelectedCurves
[02:32:49,592 | ERROR |    utils |   103]: warning() : Invalid scale factor. Reverting to default (1.0).
[02:32:49,593 | DEBUG |    utils |   105]: warning() : File "H:\Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\core.py", line 4124, in getScaleFactor
[02:32:53,855 | ERROR |    utils |   103]: warning() : Invalid scale factor. Reverting to default (1.0).
[02:32:53,856 | DEBUG |    utils |   105]: warning() : File "H:\Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\core.py", line 4124, in getScaleFactor
[02:44:04,685 | DEBUG |     main |    57]: <module>() : -----------------Starting Log Session-----------------
[02:44:04,685 | DEBUG |     main |    58]: <module>() :                  18/10/2024, 02:44:04                 
[02:44:05,476 |  INFO |     main |    85]: checkScriptJobs() : scriptJobs created for "main" UI!
[02:48:18,943 | DEBUG |     main |    57]: <module>() : -----------------Starting Log Session-----------------
[02:48:18,943 | DEBUG |     main |    58]: <module>() :                  18/10/2024, 02:48:18                 
[02:50:57,983 | DEBUG |     main |    57]: <module>() : -----------------Starting Log Session-----------------
[02:50:57,983 | DEBUG |     main |    58]: <module>() :                  18/10/2024, 02:50:57                 
[02:50:59,014 |  INFO |     main |    85]: checkScriptJobs() : scriptJobs created for "main" UI!
[02:52:18,802 | DEBUG |     main |    57]: <module>() : -----------------Starting Log Session-----------------
[02:52:18,802 | DEBUG |     main |    58]: <module>() :                  18/10/2024, 02:52:18                 
[02:52:19,680 |  INFO |     main |    85]: checkScriptJobs() : scriptJobs created for "main" UI!
[02:57:56,482 | DEBUG |     main |    57]: <module>() : -----------------Starting Log Session-----------------
[02:57:56,482 | DEBUG |     main |    58]: <module>() :                  18/10/2024, 02:57:56                 
[02:57:57,394 |  INFO |     main |    85]: checkScriptJobs() : scriptJobs created for "main" UI!
[03:00:27,335 | DEBUG |     main |    57]: <module>() : -----------------Starting Log Session-----------------
[03:00:27,335 | DEBUG |     main |    58]: <module>() :                  18/10/2024, 03:00:27                 
[03:00:28,189 |  INFO |     main |    85]: checkScriptJobs() : scriptJobs created for "main" UI!
[03:00:31,658 | ERROR |    utils |   103]: warning() : Invalid scale factor. Reverting to default (1.0).
[03:00:31,659 | DEBUG |    utils |   105]: warning() : File "H:\Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\core.py", line 4111, in getScaleFactor
[03:04:33,429 | DEBUG |     main |    57]: <module>() : -----------------Starting Log Session-----------------
[03:04:33,429 | DEBUG |     main |    58]: <module>() :                  18/10/2024, 03:04:33                 
[03:04:34,308 |  INFO |     main |    85]: checkScriptJobs() : scriptJobs created for "main" UI!
[03:10:30,304 | DEBUG |     main |    57]: <module>() : -----------------Starting Log Session-----------------
[03:10:30,305 | DEBUG |     main |    58]: <module>() :                  18/10/2024, 03:10:30                 
[03:10:31,231 |  INFO |     main |    85]: checkScriptJobs() : scriptJobs created for "main" UI!
[03:10:38,495 | ERROR |    utils |   103]: warning() : Invalid scale factor. Reverting to default (1.0).
[03:10:38,498 | DEBUG |    utils |   105]: warning() : File "H:\Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\core.py", line 4131, in getScaleFactor
[03:10:52,503 |  INFO |     core |  4055]: saveOptions() : Saving options
[03:10:54,139 |  INFO |     core |  4055]: saveOptions() : Saving options
[03:11:14,210 | ERROR |    utils |   103]: warning() : Invalid scale factor. Reverting to default (1.0).
[03:11:14,214 | DEBUG |    utils |   105]: warning() : File "H:\Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\core.py", line 4131, in getScaleFactor
[03:12:17,556 | DEBUG |     main |    57]: <module>() : -----------------Starting Log Session-----------------
[03:12:17,556 | DEBUG |     main |    58]: <module>() :                  18/10/2024, 03:12:17                 
[03:12:18,452 |  INFO |     main |    85]: checkScriptJobs() : scriptJobs created for "main" UI!
[03:12:20,006 | ERROR |    utils |   103]: warning() : Invalid scale factor. Reverting to default (1.0).
[03:12:20,012 | DEBUG |    utils |   105]: warning() : File "H:\Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\core.py", line 4131, in getScaleFactor
[03:12:20,013 | ERROR |    utils |   110]: warningInView() : Select at least two curves
[03:12:20,014 | DEBUG |    utils |   112]: warningInView() : File "H:\Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\core.py", line 629, in populate
[03:12:21,557 | ERROR |    utils |   103]: warning() : Invalid scale factor. Reverting to default (1.0).
[03:12:21,560 | DEBUG |    utils |   105]: warning() : File "H:\Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\core.py", line 4131, in getScaleFactor
[03:12:28,043 | ERROR |    utils |   103]: warning() : Invalid scale factor. Reverting to default (1.0).
[03:12:28,045 | DEBUG |    utils |   105]: warning() : File "H:\Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\core.py", line 4131, in getScaleFactor
[03:13:27,026 |  INFO |     core |  4055]: saveOptions() : Saving options
[03:13:28,359 | ERROR |    utils |   103]: warning() : Invalid scale factor. Reverting to default (1.0).
[03:13:28,360 | DEBUG |    utils |   105]: warning() : File "H:\Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\core.py", line 4131, in getScaleFactor
[03:13:35,693 | ERROR |    utils |   103]: warning() : Invalid scale factor. Reverting to default (1.0).
[03:13:35,695 | DEBUG |    utils |   105]: warning() : File "H:\Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\core.py", line 4131, in getScaleFactor
[03:13:35,695 | ERROR |    utils |   110]: warningInView() : Select at least one compatible curve (Bound/Warp curves or geometry)
[03:13:35,696 | DEBUG |    utils |   112]: warningInView() : File "H:\Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\core.py", line 1617, in unbind
[03:13:39,632 | ERROR |    utils |   103]: warning() : Invalid scale factor. Reverting to default (1.0).
[03:13:39,633 | DEBUG |    utils |   105]: warning() : File "H:\Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\core.py", line 4131, in getScaleFactor
[03:13:39,634 | ERROR |    utils |   110]: warningInView() : Select at least one compatible curve (Bound/Warp curves or geometry)
[03:13:39,634 | DEBUG |    utils |   112]: warningInView() : File "H:\Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\core.py", line 1617, in unbind
[03:13:40,416 | ERROR |    utils |   103]: warning() : Invalid scale factor. Reverting to default (1.0).
[03:13:40,418 | DEBUG |    utils |   105]: warning() : File "H:\Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\core.py", line 4131, in getScaleFactor
[03:13:40,418 | ERROR |    utils |   110]: warningInView() : Select one curve and one geometry or Select one target curve and any number of curveCards/Tubes
[03:13:40,419 | DEBUG |    utils |   112]: warningInView() : File "H:\Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\core.py", line 1359, in bind
[03:13:41,202 | ERROR |    utils |   103]: warning() : Invalid scale factor. Reverting to default (1.0).
[03:13:41,203 | DEBUG |    utils |   105]: warning() : File "H:\Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\core.py", line 4131, in getScaleFactor
[03:13:41,204 | ERROR |    utils |   110]: warningInView() : Select at least one compatible curve (Bound/Warp curves or geometry)
[03:13:41,204 | DEBUG |    utils |   112]: warningInView() : File "H:\Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\core.py", line 1617, in unbind
[03:13:44,235 | ERROR |    utils |   103]: warning() : Invalid scale factor. Reverting to default (1.0).
[03:13:44,235 | DEBUG |    utils |   105]: warning() : File "H:\Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\core.py", line 4131, in getScaleFactor
[03:13:44,236 | ERROR |    utils |   110]: warningInView() : Select at least two curves
[03:13:44,236 | DEBUG |    utils |   112]: warningInView() : File "H:\Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\core.py", line 777, in fill
[03:13:44,942 | ERROR |    utils |   110]: warningInView() : Select at least one compatible card or tube curve.
[03:13:44,943 | DEBUG |    utils |   112]: warningInView() : File "H:\Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\core.py", line 4456, in subdivideCurve
[03:18:15,527 | DEBUG |     main |    57]: <module>() : -----------------Starting Log Session-----------------
[03:18:15,546 | DEBUG |     main |    58]: <module>() :                  18/10/2024, 03:18:15                 
[03:18:16,355 |  INFO |     main |    85]: checkScriptJobs() : scriptJobs created for "main" UI!
[03:22:34,580 | DEBUG |     main |    57]: <module>() : -----------------Starting Log Session-----------------
[03:22:34,580 | DEBUG |     main |    58]: <module>() :                  18/10/2024, 03:22:34                 
[03:22:35,396 |  INFO |     main |    85]: checkScriptJobs() : scriptJobs created for "main" UI!
[03:22:41,053 | ERROR |    utils |   110]: warningInView() : Select Edges to Convert
[03:22:41,057 | DEBUG |    utils |   112]: warningInView() : File "H:\Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\core.py", line 6171, in edgeToCurve
[03:22:44,976 | ERROR |    utils |   103]: warning() : Invalid scale factor. Reverting to default (1.0).
[03:22:44,978 | DEBUG |    utils |   105]: warning() : File "H:\Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\core.py", line 4131, in getScaleFactor
[03:24:33,540 | DEBUG |     main |    57]: <module>() : -----------------Starting Log Session-----------------
[03:24:33,540 | DEBUG |     main |    58]: <module>() :                  18/10/2024, 03:24:33                 
[03:24:34,294 |  INFO |     main |    85]: checkScriptJobs() : scriptJobs created for "main" UI!
[03:24:36,746 | ERROR |    utils |   103]: warning() : Invalid scale factor. Reverting to default (1.0).
[03:24:36,748 | DEBUG |    utils |   105]: warning() : File "H:/Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\core.py", line 4131, in getScaleFactor
[03:24:51,075 | ERROR |    utils |   103]: warning() : Invalid scale factor. Reverting to default (1.0).
[03:24:51,077 | DEBUG |    utils |   105]: warning() : File "H:/Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\core.py", line 4131, in getScaleFactor
[03:24:51,078 | ERROR |    utils |   110]: warningInView() : Select at least two curves
[03:24:51,079 | DEBUG |    utils |   112]: warningInView() : File "H:/Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\core.py", line 629, in populate
[03:24:53,877 | ERROR |    utils |   103]: warning() : Invalid scale factor. Reverting to default (1.0).
[03:24:53,880 | DEBUG |    utils |   105]: warning() : File "H:/Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\core.py", line 4131, in getScaleFactor
[03:24:53,881 | ERROR |    utils |   110]: warningInView() : Select at least two curves
[03:24:53,882 | DEBUG |    utils |   112]: warningInView() : File "H:/Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\core.py", line 629, in populate
[03:25:00,799 | ERROR |    utils |   103]: warning() : Invalid scale factor. Reverting to default (1.0).
[03:25:00,802 | DEBUG |    utils |   105]: warning() : File "H:/Workspace/Art/Tools/MetaBox\Scripts\Modeling\Edit\gs_curvetools\core.py", line 4131, in getScaleFactor
=======
[14:34:57,855 | DEBUG |     main |    57]: <module>() : -----------------Starting Log Session-----------------
[14:34:57,855 | DEBUG |     main |    58]: <module>() :                  21/10/2024, 14:34:57                 
[14:34:58,672 |  INFO |     main |    85]: checkScriptJobs() : scriptJobs created for "main" UI!
[14:42:06,978 | DEBUG |     main |    57]: <module>() : -----------------Starting Log Session-----------------
[14:42:06,979 | DEBUG |     main |    58]: <module>() :                  21/10/2024, 14:42:06                 
[14:42:07,764 |  INFO |     main |    85]: checkScriptJobs() : scriptJobs created for "main" UI!
[14:57:40,174 | DEBUG |     main |    57]: <module>() : -----------------Starting Log Session-----------------
[14:57:40,175 | DEBUG |     main |    58]: <module>() :                  21/10/2024, 14:57:40                 
[14:57:40,972 |  INFO |     main |    85]: checkScriptJobs() : scriptJobs created for "main" UI!
[15:30:09,640 | DEBUG |     main |    57]: <module>() : -----------------Starting Log Session-----------------
[15:30:09,640 | DEBUG |     main |    58]: <module>() :                  21/10/2024, 15:30:09                 
[15:30:10,427 |  INFO |     main |    85]: checkScriptJobs() : scriptJobs created for "main" UI!
[15:59:32,895 | DEBUG |     main |    57]: <module>() : -----------------Starting Log Session-----------------
[15:59:32,896 | DEBUG |     main |    58]: <module>() :                  21/10/2024, 15:59:32                 
[15:59:33,700 |  INFO |     main |    85]: checkScriptJobs() : scriptJobs created for "main" UI!
>>>>>>> 7863e4b (Updated Grooms Tools)
